src/util.jl

```diff
@@ -147,6 +148,16 @@ in the other dimensions.
 function slice(dims::NTuple{N},i,j,low=1) where N
     CartesianIndices(ntuple( k-> k==j ? (i:i) : (low:dims[k]), N))
 end
+"""
+    removeMean!(s)
+
+Remove the mean of scalar array `s` in physical domain from `s`.
+This is potentially useful for pressure normalization.
+"""
+function removeMean!(s)
```

Comment:
Why do we need to add this?

Reply:
This function is for pressure nomalization in runtests.jl. I can also leave it in runtest.jl. Just in case we are going to use it at some point in the future.

Reply:
Even better would be to write a test that doesn't need a new function.